### ISSUE TITLE
bro wtf

### ISSUE THREAD

**Issue by MuffetYMuff** [luckydog7/Funkin-android#691]
Hey Bro Wtf U Said We Fixed Black Sprites Bug But İ Have A Screenshot Look At This (Update) İ Can't Enter Game Now Because Game Starting With Black Screen And İ Can't Play 
![Screenshot_2021-05-05-22-05-08-023_com ninjamuffin99 funkin](https://user-images.githubusercontent.com/83839339/117497828-a6dbd980-af81-11eb-917c-6f16203c868c.jpg)

**Comment by FridayNightSans** [comment]
Idk, ask luckydog ;-;

**Comment by MuffetYMuff** [comment]
> Lucky Dog did put out, "a lil bit" on the update

yes he said but i can't play game now!! everything is black now!!!

**Comment by hrulol** [comment]
u need less ram the less ram u have the more ur phone/tablet can handle.
